const handleError=(err,req,res,next)=>{
    console.log("error here",err)
}
module.exports={
    handleError
}